src/Model/EventModel.php

```diff
@@ -44,9 +44,11 @@ public function __construct(array $config ) {
         $this->properties = $config['properties'];
         // Can pass in unix timestamp if prefixed with '@' or any date/time format accepted by DateTime interface.
         try {
-            $time = (array)new DateTime(
-                is_int($config['time']) ? '@' . $config['time'] : $config['time']
-            );
+            if (isset($config['time'])) {
+              $time = new DateTime(
+                  is_int($config['time']) ? '@' . $config['time'] : $config['time']
+              );
```

Comment:
Super nit: looks like these three lines need an additional indentation of 2 spaces